### DISCUSSION TITLE
A Plague Tale: Innocence & Requiem | Can not make them work, on the latest Wine + DXVK + GPTK

### DISCUSSION THREAD

**Discussion by ferreira-ff** [The-Wineskin-Project/WineskinServer#438] [Q&A] [answered]
Hello. Recently I bought both **A Plague Tale: Innocence and A Plague Tale: Requiem** from GOG.
With the help of **WineSkin**, I have made the most recent "**MyCoolWrapper(s)**" of _Wine + DXVK_ and _Wine + GPTK_.

"Innocence" **DX11** crashes when I select start on the game's menu; "Requiem" **DX12** stays in a black screen. I have tried every trick, in the book.

Curiously; both games **install and work** on _"CrossOver + GPTK patch"_. I have tried to compare, the **bottles and the wrappers**; I am completely out of ideas...

What am I forgetting here? Any help would be really appreciated.

Kind regards to all

**Comment by Gcenx** [comment, marked as answer]
You shouldn’t be mixing DXVK & D3DMetal.

CXPatcher has a rather nasty hack that allows it to somewhat work as CrossOver DXVK toggle removes the files when it’s disabled and copied then in again when it’s enabled.

Also the current wrapper may have issues if you have a brew Intel gstreamer installation, that’s already resolved for the next wrapper but holding off until the next MoltenVK release.

**Reply by ferreira-ff** [reply]
Hello, my friend. I am glad that it is not a user error; thank you for the information, regarding the Crossover "patch".

I already erased the Application, the related files on my ~/Library and reinstalled CrossOver default. I am only testing; I do not know if buying the license, the money goes to the Wine developers.

Kind regards

**Reply by Gcenx** [reply]
CodeWeavers employ a large portion on wine developers so having a license literally pays these developers.

I’m personally one of the upstream Winehq macOS package maintainers and I’ve been involved in many other “wine” wrapper projects.

I send feedback to CodeWeavers and upstream wine along with opening bugs etc, these same can’t be said for other projects though I guess nobody cares anymore.
